content/en/guides/hosting/iam/scim.md

```diff
@@ -1173,6 +1424,128 @@ POST /scim/Roles
 }
 ```
 
+### Update custom role
+
+#### Add permissions to role
+
+- **Endpoint**: **`<host-url>/scim/Roles/{id}`**
+- **Method**: PATCH
+- **Description**: Add permissions to an existing custom role.
+
+{{< tabpane text=true >}}
+{{% tab header="Request" %}}
+```bash
+PATCH /scim/Roles/{role_id}
+```
+
+```json
+{
+    "schemas": ["urn:ietf:params:scim:api:messages:2.0:PatchOp"],
+    "Operations": [
+        {
+            "op": "add",
+            "path": "permissions",
+            "value": [
+                {
+                    "name": "project:delete"
+                },
+                {
+                    "name": "run:stop"
+                }
+            ]
+        }
+    ]
+}
+```
+{{% /tab %}}
+{{% tab header="Response" %}}
+```bash
+(Status 200)
+```
+
+Returns the updated role with new permissions added.
+{{% /tab %}}
+{{< /tabpane >}}
+
+#### Remove permissions from role
+
+- **Endpoint**: **`<host-url>/scim/Roles/{id}`**
+- **Method**: PATCH
+- **Description**: Remove permissions from an existing custom role.
+
+{{< tabpane text=true >}}
+{{% tab header="Request" %}}
+```bash
+PATCH /scim/Roles/{role_id}
+```
+
+```json
+{
+    "schemas": ["urn:ietf:params:scim:api:messages:2.0:PatchOp"],
+    "Operations": [
+        {
+            "op": "remove",
+            "path": "permissions",
+            "value": [
+                {
+                    "name": "project:delete"
```

Comment:
It's confusing that we are _removing_ a permission to _delete_, maybe choose a different permission for this example.